sagemaker-experiments/tensorflow2-california-housing-regression-experiment/.gitignore

```diff
@@ -0,0 +1 @@
+data
```

Comment:
minor: remove the .gitignore?

Reply:
why? the notebook downloads a bunch of data, and this excludes it from git operations

Reply:
makes sense, it just didn't seem related to the other new changes so wasn't sure if it was intentional

Reply:
makes sense, it just didn't seem related to the other new changes so wasn't sure if it was intentional